### PR TITLE
New tooltip woooho

### PR DESCRIPTION
**Pull request by geiger01**

### CODE REVIEW

File: packages/nindo/src/external/components/tooltip/tooltip.comp.tsx [hunk 1]
```diff
@@ -7,19 +8,19 @@ type TooltipProps = {
...
-				className={`tooltip-content ${direction || 'top'}`}
-				style={{ width: `${width || 200}px` }}
-				dangerouslySetInnerHTML={{ __html: content }}
-			></span>
-		</span>
+		<>
+			<span className="tooltip" data-tip={content} >
+				<span className="tooltip-pointer">{pointer || '?'}</span>
+			</span>
+			<ReactTooltip textColor={textColor || 'white'} place={direction || "top"} backgroundColor={backgroundColor || '#ff4572'} className={`react-tooltip`} html={true} effect="solid" />
```
Comment (dsternlicht) [inline packages/nindo/src/external/components/tooltip/tooltip.comp.tsx:23]: I think the the default bg color should be black. The direction shouldn't be dynamic?

File: packages/nindo/src/external/components/tooltip/tooltip.scss [hunk 2]
```diff
@@ -1,5 +1,10 @@
 @import '../../styles/_vars';
 
+.react-tooltip {
+	max-width: 300px;
+	opacity: 1 !important;
```
Comment (dsternlicht) [inline packages/nindo/src/external/components/tooltip/tooltip.scss:5]: Why important?

File: packages/nindo/src/external/components/tooltip/tooltip.comp.tsx [hunk 3]
```diff
@@ -7,19 +8,19 @@ type TooltipProps = {
 	direction?: 'bottom' | 'left' | 'right' | 'top';
 	pointer?: string;
 	width?: number;
+	backgroundColor?: string
+	textColor?: string
```
Comment (dsternlicht) [inline packages/nindo/src/external/components/tooltip/tooltip.comp.tsx:12]: Maybe we should have `pointerStyles` as well for changing the pointer?

**Review by dsternlicht — APPROVED** [review]

File: packages/nindo/src/external/components/tooltip/tooltip.comp.tsx (hunk 1 above)
Comment (geiger01) [reply packages/nindo/src/external/components/tooltip/tooltip.comp.tsx:23]: The direction is dynamic, the default is top but it changes according to the window space

**Review by dsternlicht — APPROVED** [review]

**Pull request merged**